gapic-common/lib/gapic/operation.rb

```diff
@@ -224,8 +225,14 @@ def delete options: nil
     #
     def reload! options: nil
       # Converts hash and nil to an options object
-      options = Gapic::CallOptions.new(**options.to_h) if options.respond_to? :to_h
-
+      options = if options.respond_to? :to_h
+                  options.to_h
+                else
+                  {}
+                end
+      options.merge! @options.to_h
```

Comment:
This could modify the incoming argument if a hash is passed in. Can you just do something like:

```ruby
options = if options.respond_to? :to_h
            options.to_h.merge @options.to_h
          else
            @options.to_h
          end
```